animalfacts.py

```diff
@@ -1825,6 +1827,7 @@ def animalfactsbot(reddit):
     )
 
 MINK_FACTS = (
+	'Mink is a mammal that belongs to the family of weasels.',
```

Comment:
Should really be "The mink" if you could?

Reply:
Corrected!